homework/homework_test.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"fmt"
+	"gocourse05/core/camera"
+	"strings"
+	"testing"
+)
+
+var lastSentData string
+
+func mockSendDataToServer(data map[int]string) {
+	dataToServer := ""
+	for _, v := range data {
+		dataToServer += fmt.Sprintf("{%s}, ", v)
+	}
+
+	lastSentData = dataToServer
+}
+
+func TestMainFunctionSuccessful(t *testing.T) {
```

Comment:
Напевно `Function` зайве. Кожен тест тестує функцію.
```suggestion
func TestMainSuccessful(t *testing.T) {
```

Reply:
так, але не актульно